Formula/nemu.rb

```diff
@@ -0,0 +1,31 @@
+class Nemu < Formula
+  desc "Ncurses UI for QEMU"
+  homepage "https://github.com/nemuTUI/nemu"
+  url "https://github.com/nemuTUI/nemu/archive/refs/tags/v3.2.0.tar.gz"
+  sha256 "f35b7e5c534bae294fe6df7685b686682145222a954b6914dfdcc17d386fecba"
+  license "BSD-2-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "gettext"
+  depends_on "json-c"
+  depends_on "libarchive"
+  depends_on "ncurses"
+  depends_on "openssl@3"
+  depends_on "pkg-config"
```

Comment:
Why are `gettext` and `pkg-config` runtime dependencies?

Reply:
gettext is a runtime dependence. It used for localization in run time. pkg-config is not, will fix it.

Reply:
Fixed

Reply:
Update2 Still gettext runtime dependency
`/opt/homebrew/opt/gettext/lib/libintl.8.dylib (gettext)`

Reply:
I see, that is good to know.